src/ports/postgres/modules/deep_learning/madlib_keras_wrapper.py_in

```diff
@@ -287,27 +298,50 @@ def parse_optimizer(compile_dict):
 
 
 # Parse the fit parameters into a dictionary.
-def parse_and_validate_fit_params(fit_param_str):
+def parse_and_validate_fit_params(fit_param_str, current_seg_id=0):
```

Comment:
We call this function in two contexts:
1) During the fit transition function for parsing: We have the segment_id
2) During validation: We don't have the segment_id since it is running on master. 

We can pass a value instead of having a default but it will be the exact same constant in any case.

Reply:
I see makes sense. So when it is running on master, maybe we should use -1 instead of 0

Reply:
I like -1 better as well... if someone is trying to debug a problem and they see segment_id=-1 printed out, that will be more helpful and less prone to confusion than if it shows up as segment_id=0